packages/backend/src/accounting/psql/service.test.ts

```diff
@@ -108,33 +108,33 @@ describe('Psql Accounting Service', (): void => {
     })
   })
 
-  describe('createSettlementAccount', (): void => {
+  describe('createLiquidityAndLinkedSettlementAccount', (): void => {
     test('creates account', async (): Promise<void> => {
```

Comment:
let's have a test to make sure the settlement account was also created

Reply:
I agree. Done.